source/agora/cli/checkvtable/generate.d

```diff
@@ -0,0 +1,318 @@
+/*******************************************************************************
+
+    The agora-checkvtable-gen for generating C++ source code
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.cli.checkvtable.generate;
+
+import agora.cli.checkvtable.types;
+
+import std.file;
+import std.format;
+import std.stdio;
+
+version (Windows) { pragma(msg, "This does not support Windows."); } else:
```

Comment:
What was wrong with checking the vtable on Windows?

Reply:
> What was wrong with checking the vtable on Windows?

I tested 'MS Visual C++' on the following site.

Reply:
If the offset of vtable's virtual methods is proven to be the same in `gcc`, then of course in Windows, we no longer have to make unnecessary efforts.
The test will also be run only on CI.